examples/apps/collaborative-textarea/README.md

```diff
@@ -17,8 +16,8 @@ You can run this example using the following steps:
 1. Run `pnpm install` and `npm run build:fast -- --nolint` from the `FluidFramework` root directory.
     - For an even faster build, you can add the package name to the build command, like this:
       `npm run build:fast -- --nolint @fluid-example/collaborative-textarea`
-1. In a separate terminal, start a Tinylicious server by following the instructions in [Tinylicious](../../../server/tinylicious).
-1. Run `npm start` from this directory (examples/apps/collaborative-textarea) and open <http://localhost:8080> in a web browser to see the app running.
+1. In a separate terminal, start a Tinylicious server by following the instructions in [Tinylicious](https://github.com/microsoft/FluidFramework/tree/main/server/tinylicious).
```

Comment:
That's an interesting point, which I hadn't considered. Since I am hoisting this logic into a stand-alone-ish library, I tried to remove any assumptions about where various files exist in the repo. In theory this has a portability benefit, but it makes it hard to do things like link to specific files elsewhere in the repo.

My inclination is that the portability win outweighs the benefit of keeping the relative file path for such a case, but that's only my opinion. Definitely interested in your thoughts there.